README.md

```diff
@@ -28,12 +28,41 @@
 ## Documentation
 
 - [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/v0.10.0.rc5)
-  - [Guides](docs)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/v0.10.0.rc5)
+  - [Guides](docs)      
+    - [Getting Started](docs/general/getting_started.md)
+    - [Configuration Options](docs/general/configuration_options.md)
+    - [Serializers](docs/general/serializers.md)
+    - [Adapters](docs/general/adapters.md)
+    - [Rendering](docs/general/rendering.md)
+    - [Caching](docs/general/caching.md)
+    - [Logging](docs/general/logging.md)
+    - [Deserialization](docs/general/deserialization.md)
+    - [Instrumentation](docs/general/instrumentation.md)
+    - JSON API
+      - [Schema](docs/jsonapi/schema.md)
+      - [Errors](docs/jsonapi/errors.md)
+    - How To
+      - [How to add root key](docs/howto/add_root_key.md)
+      - [How to add pagination links](docs/howto/add_pagination_links.md)
+      - [Using ActiveModelSerializers Outside Of Controllers](docs/howto/outside_controller_use.md)
+      - [Testing ActiveModelSerializers](docs/howto/test.md)
+      - [Passing Arbitrary Options](docs/howto/passing_arbitrary_options.md)
+      - [How to serialize a Plain-Old Ruby Object (PORO)](docs/howto/serialize_poro.md)
+      - [Integrations with Ember, Grape and Sinatra](docs#integrations)
+
+-----------------------------------------------------
+
+### Older Documentation
+
 - [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
+
 - [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+
+
+-------------------------------------------------------
 
 ## About
```

Comment:
The about section?

I think we may want a separate PR for the about section, cause it's kinda long. I'd like to shrink it a bit / link to other places for more information on stuff like each major piece (Adapter, Serializer, etc)